pippo-test/pom.xml

```diff
@@ -33,11 +35,23 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
```

Comment:
We don't need this dependency in pippo-test module. This module is used also by users that go with other web server (tomcat, undertow)

Reply:
This dependency can be moved in `pippo-jetty` module and added `test` scope.

Reply:
As previously discussed, instead of moving WebSocket test **client** to another project, I've extracted WebSocket-related test infrastructure from `PippoExtension` to `PippoWebSocketExtension` and made `websocket-jetty-client` dependency optional.